clang/include/clang/Basic/DiagnosticParseKinds.td

```diff
@@ -268,6 +268,8 @@ def err_expected_semi_after_namespace_name : Error<
   "expected ';' after namespace name">;
 def err_unexpected_namespace_attributes_alias : Error<
   "attributes cannot be specified on namespace alias">;
+def err_unexpected_qualified_namespace_alias : Error<
+  "unexpected nested name specifier in namespace alias definition">;
```

Comment:
I’m not sure how many people would know what a ‘nested name specifier’ is—or at least it requires a bit of thinking. Maybe something like ‘namespace alias must be a single identifier’ would be a bit more obvious?

Reply:
Sounds good - I was trying to think of the right phrasing and had found some references to "nested name specifiers" in other diagnostics. But I like your suggested phrasing 👍